data/json/items/obsolete.json

```diff
@@ -517,5 +517,27 @@
         "volume": 1,
         "bashing": 4,
         "to_hit": -2
+    },
+    {
+        "id": "jar_V8",
```

Comment:
Is it possible to have a migration that creates nested items or will I need to write it myself?